content/integrations/bugsnag.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Datadog-Bugsnag Integration
+integration_title: Bugsnag
+kind: integration
+doclevel: basic
+---
+
+### Overview
+
+Bugsnag provides software teams with an automated crash detection platform for their web and mobile applications. Bugsnag automatically captures and alerts you of errors as they happen. Integrating Datadog with Bugsnag will send error notifications to your Datadog event stream.
+
+With this integration:
+
+- Get a summary of the error in your Datadog event stream
+- Get notified when a project has a spike in error rates
+- Filter notifications by severity and release stage
+
+### Installation
+
+No installation is required.
+
+### Configuration
+
+To integrate Bugsnag with Datadog:
+
+1. In [Bugsnag](https://bugsnag.com/) go to _Settings_ for the project you would like to configure to send notifications to Datadog
+1. Select _Team Notifications_ and then _Datadog_
+1. Customize the notifications you'll see in Datadog by selecting error notification triggers.
+1. Apply custom filters to your notification triggers to see errors from specific release stages and severities.
+1. Enter your Datadog API key.
+1. Select _Test Notification_ to test the configuration. A test error from Bugsnag will appear in Datadog's event stream.
+1. _Save_ your settings.
+1. Add more streams from the same project to see error events based on a different set of notification criteria.
+
```

Comment:
"1. Select _Test Notification_ to test the configuration. A test error from Bugsnag will appear in Datadog's event stream."